requirements.txt

```diff
@@ -1,2 +1,3 @@
-pandas
+pandas==1.5.1
 git+https://github.com/nestauk/nesta_ds_utils.git
```

Comment:
I just tried to pip install from this branch and got this error:

"error in dap_aria_mapping setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Parse error at "'+https:/'": Expected string_end"

Is this the issue you were mentioning about installing the nesta_ds_utils package?

Reply:
I've amended this line so it works - but this isn't the error I was getting, I was getting things like this `ERROR: Could not find a version that satisfies the requirement numpy==1.23.4 (from nesta-ds-utils)`